packages/Text/Text.jsx

```diff
@@ -17,6 +17,7 @@ const Text = ({ block, bold, size, invert, children, ...rest }, context) => {
   const textColor = invert ? styles.invertedColor : styles.color
 
   const classes = joinClassNames(
+    styles.default,
```

Comment:
Be mindful of my other comment in **typography.modules.scss** related to this class.